dask/dataframe/io/tests/test_parquet.py

```diff
@@ -632,7 +631,8 @@ def test_append_with_partition(tmpdir, engine):
     df1.index.name = "index"
     dd_df0 = dd.from_pandas(df0, npartitions=1)
     dd_df1 = dd.from_pandas(df1, npartitions=1)
-    dd.to_parquet(dd_df0, tmp, partition_on=["lon"], engine=engine)
+    fut = dd.to_parquet(dd_df0, tmp, partition_on=["lon"], engine=engine, compute=False)
+    fut.compute(scheduler="single-threaded")
```

Comment:
Was this just for debugging? Can it be reverted now?

Reply:
Thanks - forgot about these debug changes :)